invenio_assets/assets/build/webpack.config.js

```diff
@@ -25,9 +35,7 @@ var webpackConfig = {
     symlinks: false,
     alias: {
       "@templates": path.resolve(config.build.context, "templates"),
-      // Semantic-UI LESS config resolving alias
-      // TODO: This would be better injected via config from Flask...
-      "../../theme.config$": path.resolve(config.build.context, "less/invenio_theme/theme.config"),
+      ...aliases,
```

Comment:
I would remove the `@templates` now, and move it to a Invenio RDM config

Reply:
Created https://github.com/inveniosoftware/invenio-app-rdm/issues/126